ee/clickhouse/migrations/0025_remove_dollar_from_materialized_columns.py

```diff
@@ -0,0 +1,74 @@
+from functools import lru_cache
+
+import structlog
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.client import sync_execute
+from ee.clickhouse.materialized_columns.columns import get_materialized_columns, materialized_column_name
+from posthog.constants import GROUP_TYPES_LIMIT
+from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_DATABASE
+
+logger = structlog.get_logger(__name__)
+
+
+def alias_column(table, current_name, new_name):
+    sync_execute(
+        f"""
+        ALTER TABLE {table} ON CLUSTER '{CLICKHOUSE_CLUSTER}'
+        ADD COLUMN IF NOT EXISTS \"{new_name}\"
+        VARCHAR ALIAS \"{current_name}\"
```

Comment:
Is it always varchar? I guess we treat everything as a string then interpret on read if we need to?

Reply:
Materialized columns are all varchars currently yeah. :)